iac-extensions/arm/src/main/java/org/sonar/iac/arm/checks/ClearTextProtocolsCheck.java

```diff
@@ -45,6 +45,6 @@ private static void checkHttpsFlag(ContextualResource resource) {
 
   private static void checkFtpsState(ContextualResource resource) {
     resource.property("ftpsState")
-      .reportIf(isValue("AllAllowed"), GENERAL_ISSUE_MESSAGE);
+      .reportIf(isEquals("AllAllowed"), GENERAL_ISSUE_MESSAGE);
```

Comment:
I think the `s` is grammatically not correct. It should be `isEqual` or `equals`